src/components/layout/tables/AntTable/AntTable.tsx

```diff
@@ -0,0 +1,86 @@
+import React from 'react';
+
+import { useIsMobile } from '../../../../hooks';
+import {
+  BookmarkIcon,
+  BorderOuterIcon,
+  FileCodeIcon,
+  RefreshAlertIcon,
+} from '../../../icons';
+import AntRow from '../AntRow/AntRow';
+import './styles.css';
+
+function AntTable({ antIds }: { antIds: string[] }) {
+  const isMobile = useIsMobile();
+
+  return (
+    <div className="flex-column center">
+      <table className="assets-table">
+        <thead>
+          <tr className="assets-table-header">
+            <td className="assets-table-item">
+              Nickname&nbsp;
+              <BookmarkIcon
+                width={'20px'}
+                height={'30px'}
+                fill={'var(--text-faded)'}
+              />
+            </td>
+            <td className="assets-table-item center">
+              Contract ID&nbsp;
+              <FileCodeIcon
+                width={'20px'}
+                height={'30px'}
+                fill={'var(--text-faded)'}
+              />
+            </td>
+            {isMobile ? (
+              <></>
+            ) : (
+              <td className="assets-table-item center">
+                Target ID&nbsp;
+                <BorderOuterIcon
+                  width={'20px'}
+                  height={'30px'}
+                  fill={'var(--text-faded)'}
+                />
+              </td>
+            )}
+            <td className="assets-table-item center">
+              Status&nbsp;
+              <RefreshAlertIcon
+                width={'20px'}
+                height={'30px'}
+                fill={'var(--text-faded)'}
+              />
+            </td>
+            {isMobile ? <></> : <td className="assets-table-item"></td>}
+          </tr>
+        </thead>
+        <tbody
+          className="flex-column center"
+          style={{ gap: '.5em', minHeight: 200 }}
+        >
+          {antIds.map((id, index) =>
+            React.cloneElement(
+              <AntRow
+                antId={id}
+                bgColor={'#1E1E1E'}
+                textColor={'var(--text-white)'}
+              />,
+              { key: index },
```

Comment:
we don't need to clone here, you can return the AntRow component directly from the map

Reply:
and add key to the props